esx_service/cli/vmdkops_admin.py

```diff
@@ -955,9 +958,13 @@ def tenant_access_set(args):
     if args.volume_totalsize:
         volume_totalsize_in_MB = convert.convert_to_MB(args.volume_totalsize)
 
+    if not validate_create_value(str(args.allow_create)):
```

Comment:
1. We have two different interfaces: VMODL API and Admin CLI - both will call auth_api methods. So we should add the validation logic in auth_api, so that we don't need to repeat the logic in two different interfaces.

2. We should add the validation logic in both tenant_access_add and tenant_access_set methods.